### PR TITLE
Fixed a bug where the sound distance wasnt being recorded properly

### PR DESCRIPTION
**Pull request by rojtekin**

### CODE REVIEW

**Review by pkasinski — APPROVED** [review]
LGTM

File: core/src/model/PlayerCharacter.java [hunk 1]
```diff
@@ -51,14 +40,31 @@ public void weaponAttack(int rotationStart, int rotationFinish, int animationpar
     public PlayerCharacter(float spawnX, float spawnY, Faction faction, World<IEntity> world) {
         super(spawnX, spawnY, 32, 32, 5, 10, 0, faction, world);
         weapon = new Sword(world);
+        weapon = new Sword(world);
```
Comment (Suedie) [inline core/src/model/PlayerCharacter.java]: Weapon created twice.

**Review by Suedie — APPROVED** [review]
LGTM

File: core/src/model/PlayerCharacter.java (hunk 1 above)
Comment (rojtekin) [reply core/src/model/PlayerCharacter.java]: nice catch will fix

**Pull request merged**